src/style.css

```diff
@@ -1,43 +1,243 @@
+html {
+  box-sizing: border-box;
+  font-family: 'Roboto Slab', serif;
+}
+
 body {
-  margin: 0;
-  padding: 0;
+  font-family: 'Roboto Slab', serif;
 }
 
-.menu-list {
+li {
   list-style-type: none;
-  width: 100%;
+  padding: 10px;
+}
+
+ul {
+  padding-left: 0;
+}
+
+.bg-blue {
+  background-color: #0290ff;
+}
+
+.book-page {
+  display: flex;
```

Comment:
- [ ] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used 13 times in the CSS. Please refactor it to a class and apply the class to the JSX elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to JSX element that needs this behavior:

```JSX
  <section className="d-flex">
```